css/lodesiempre.html

```diff
@@ -1,14 +1,12 @@
 <!DOCTYPE html>
-<html>
+<html >
 
 <head>
```

Comment:
let's add the `<meta charset="utf-8">` to the header, so browsers can detect the encoding (that was what was happening on Safari)